core/lib/constants.ts

```diff
@@ -1,7 +1,25 @@
+import { paths } from "jsonpath";
+import { resolve } from "vscode-languageserver/lib/files";
```

Comment:
Looks like these got added above the copyright header, could you move them down?